conda_forge_tick/cli.xsh

```diff
@@ -26,13 +26,27 @@ def deploy(args):
             @(cmd)
         except Exception as e:
             print(e)
-    doctr_run(
-        ['git',
-         'push',
-         'https://{token}@github.com/{deploy_repo}.git'.format(
-             token=$PASSWORD, deploy_repo='regro/cf-graph-countyfair'),
-         'master'],
-         token =$PASSWORD.encode('utf-8'))
+
+    status = 1
+    try = 0
+    while status != 0 and try < 10;
+        try:
+            @(['git', 'pull', '-s', 'recursive', '-X', 'theirs'])
+        except Exception:
+            pass
+        status = doctr_run(
+            ['git',
+             'push',
+             'https://{token}@github.com/{deploy_repo}.git'.format(
+                 token=$PASSWORD, deploy_repo='regro/cf-graph-countyfair'),
+             'master'],
+             token =$PASSWORD.encode('utf-8'))
+
+        try += 1
+
+    if status != 0:
+        raise RuntimeError("bot did not push its data! stopping!")
```

Comment:
push it to a branch?

Reply:
good idea

Reply:
Also maybe halt itself in the circle_worker repo? (and ping us on all the commits so we can't ignore it?)

Reply:
I did that. Now the restart script only runs if deploy works

Reply:
cool

Reply:
done